rosetta/create-distribution.sh

```diff
@@ -74,25 +84,26 @@ git branch --force distribution-base $DISTRIBUTION_BASE_REF
 git branch --force rosetta-distribution distribution-base
 git switch rosetta-distribution
 
-##############################
-# Update NV Internal patches #
-##############################
+####################################
+# Branch patches from a local repo #
+####################################
 # Create local branches for all remote branches since remotes aren't recognized
 # when adding local repos as remotes. Excludes origin/HEAD and origin/test_\d+
-function git-nv-t5x {
-  git -C $INTERNAL_T5X_SUBMODULE_DIR "$@"
-}
-NV_REMOTE_NAME=nv-t5x
-if [[ -d $INTERNAL_T5X_SUBMODULE_DIR ]]; then
-  for remote_branch in $(git-nv-t5x branch --remotes | grep -v 'origin/HEAD' | egrep -v 'origin/test_[0-9]+'); do
-    # Try creating a local tracking branch, but if already exists, then update it to match remote
-    git-nv-t5x checkout --track $remote_branch || git-nv-t5x reset --hard $remote_branch
+if [[ -n "${EXTRA_MIRROR_DIR+x}" ]] && [[ -d $EXTRA_MIRROR_DIR ]]; then
+  EXTRA_REMOTE_NAME=extra
+  function git-extra {
```

Comment:
Since this command appends an additional mirror to the git command, would `git+mirror` be more intuitive?